ArduCopter/mode_loiter.cpp

```diff
@@ -147,7 +147,7 @@ void ModeLoiter::run()
     case AltHold_Landed_Pre_Takeoff:
         attitude_control->reset_rate_controller_I_terms_smoothly();
         loiter_nav->init_target();
-        attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(0.0f, 0.0f, 0.0f);
```

Comment:
You've probably already checked for other instances of input_euler_angle_roll_pitch_euler_rate_yaw(0.0f, 0.0f, 0.0f); but in any case I see we are still using this in:

- mode_throw.cpp
- takeoff.cpp
- Mode::zero_throttle_and_relax_ac
- Mode::make_safe_spool_down